docs/external-listener/index.md

```diff
@@ -57,6 +67,13 @@ To configure an external listener that uses the LoadBalancer access method, comp
           externalStartingPort: 19090
           containerPort: 9094
           accessMethod: LoadBalancer
+          # set which port clients can use to reach all the brokers of the Kafka cluster, default is 29092
+          # valid range: 0 < x < 65536
+          anyCastPort: 443
+          # set which port the ingress controller uses to handle the external client traffic through the "anyCastPort", default is 29092
+          # valid range: 1023 < x < 65536
+          # if specified, the ingressControllerTargetPort cannot collide with the reserved envoy ports (if using envoy) and the external broker port numbers
+         ingressControllerTargetPort: 3000
```

Comment:
**Note**: the comments should be identical to the ones above.

Reply:
Updated